website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
Huh, interesting. I'm not super-familiar with `require` and I didn't know you could require CSS files. Can I ask where the `require` global comes from? Are we using [RequireJS](http://requirejs.org/)? How does that end up in our global javascript namespace?